simplelogger.ino

```diff
@@ -0,0 +1,165 @@
+#include <SPI.h>
+#include <SD.h>
+#include <DHT_U.h>
+#include <DHT.h>
+
+///////////////////////////////////////
+// Change filename and sensor type here 
+const String LOG_FILENAME = "dhtlog.csv";
+#define DHT_TYPE DHT11
+
+///////////////////////////////////////////
+// Pin selection
+// Pins 11, 12, and 13 (on arduino uno) are required for the SPI for the SD card reader, so don't use them.
+#define CS_PIN 4 // cs pin for sd card
+#define DHT_PIN 5 // data pin for dht sensor
+#define LED_PIN 8 // led pin for indicator. not required.
+
+
+class DHTLogger
```

Comment:
When would the `simplelogger.ino` file be used instead of `sensorlogger.ino`? Just for testing?